src/ignitus-HomePage/ignitus-GetStarted/Components/index.tsx

```diff
@@ -19,9 +19,17 @@ export const GetStarted: React.FunctionComponent = withErrorBoundary(() => (
       </Paragraph>
     </TopSection>
     <BottomSection>
-      <Button size="large" category="secondary">
-        <Link to="/Signup">Sign-Up</Link>
-      </Button>
+      { localStorage.getItem('authenticated')? 
+        <a href = "https://app.slack.com/client/TRN1H1V43/CUCLB9VJL" target = "_blank">
+          <Button category="primary" size="large">
```

Comment:
Please change the category of a button from `primary` to `secondary` here.

Reply:
Done

Reply:
I still don't see the change. 
<img width="568" alt="Screenshot 2020-05-02 at 7 07 48 PM" src="https://user-images.githubusercontent.com/26546816/80870667-44823600-8ca8-11ea-8c62-de9dbf325951.png">

Reply:
Sorry, I must've missed it. Updated.